Src/EB/AMReX_EB2_3D_C.cpp

```diff
@@ -76,10 +76,10 @@ void set_eb_data (const int i, const int j, const int k,
         return;
     }
 
-    Real dapx = axm - axp;
-    Real dapy = aym - ayp;
-    Real dapz = azm - azp;
-    Real apnorm = std::sqrt(dapx*dapx+dapy*dapy+dapz*dapz);
+    Real dapx = (axm - axp)*(dx[1]*dx[2]);
+    Real dapy = (aym - ayp)*(dx[0]*dx[2]);
+    Real dapz = (azm - azp)*(dx[0]*dx[1]);
+    const Real apnorm = std::sqrt(dapx*dapx+dapy*dapy+dapz*dapz) + 1.e-30_rt*std::sqrt(dx[0]*dx[1]*dx[2]);
```

Comment:
The second term has a different units than the first. The first term is L^2, whereas the second is L^(3/2).